### ISSUE TITLE
Please remove this title on the blank page.

### ISSUE THREAD

**Issue by Mesum-Hussain** [TeamNewPipe/NewPipe#7610]

**Comment by SameenAhnaf** [comment]
Is blank page required at all? If any dev wants to maintain privacy while screen recording, he or she might just use any of the kiosk pages.

**Comment by Mesum-Hussain** [comment]
> Is blank page required at all? If any dev wants to maintain privacy while screen recording, he or she might just use any of the kiosk pages. 

I am using blank page for avoiding home page feeds in order to get de-addicted from watching mindless video. I think blank page is a very useful feature of NewPipe and it shouldn't be removed.

**Comment by ChiefMikeK** [comment]
![BlankPage](https://user-images.githubusercontent.com/20070053/147874501-5f6e201e-53d6-40a9-80f3-6e47fb82854b.jpg)
No need to have NewPipe display twice

**Comment by ChiefMikeK** [comment]
For example using U+21EA  
`<string name="main_bg_subtitle">Tap the magnifying glass ⇪\n to get started!</string>`

**Comment by mhmdanas** [comment]
@ChiefMikeK I don't know, I don't think the arrow is necessary. However, I'm not sure about your other suggestions yet.

**Comment by Mesum-Hussain** [comment]
> For example using U+21EA  
> `<string name="main_bg_subtitle">Tap the magnifying glass ⇪\n to get started!</string>`
> 

wouldn't be leaving the "blank Page" blank, a good idea? Why do we need any text in the first place? It's great for first time use to know how to get started (if you don't have enough common sense to click the magnifying glass for searching without being asked to), but it gets annoying every time you open the app and there is a big heading text telling you what to do as if you are dumb. It's a bad UX in my opinion.

**Comment by yashpalgoyal1304** [comment]
i think this issue can be closed

**Comment by Mesum-Hussain** [comment]
> i think this issue can be closed

Maybe.....but what's the conclusion btw? 
Will the text remain unchanged or will the **blank page** be made actually blank?

**Comment by ChiefMikeK** [comment]
https://github.com/TeamNewPipe/NewPipe/blob/dev/app/src/main/res/values/strings.xml#L3
Trivial change most likely forgotten `¯\_(ツ)_/¯` also needs some discussion.   
> close as `won't fix` or ???
@OxygenCobalt since you are the last person that changed this file any comment?

**Comment by opusforlife2** [comment]
Yeah that text looks like from back when the blank page _was_ the default tab. Since it now needs to be added manually, the help text isn't needed anymore.

**Comment by Mesum-Hussain** [comment]
> Yeah that text looks like from back when the blank page _was_ the default tab. Since it now needs to be added manually, the help text isn't needed anymore.

So would it be fine if I remove the text?
I will make the changes tomorrow if it's fine.

**Comment by opusforlife2** [comment]
Sure, please go ahead.
